redash/settings.py

```diff
@@ -90,6 +90,27 @@ def all_settings():
 SAML_LOGIN_ENABLED = SAML_METADATA_URL != ""
 SAML_CALLBACK_SERVER_NAME = os.environ.get("REDASH_SAML_CALLBACK_SERVER_NAME", "")
 
+# Enables the use of an externally-provided REMOTE_USER environment var.
+# REMOTE_USER should be set to an email address
+#
+# Note that redash will only check REMOTE_USER once, upon the first need for a
+# login, and then set a cookie which keeps the user logged in.  Dropping
+# REMOTE_USER after subsequent requests won't automatically log the user out.
+# Doing so could be done with further work, but usually it's unnecessary.
+#
+# In proxy situations, it may be desirable to forward REMOTE_USER via a HTTP
+# request header, which you can specify by setting REDASH_REMOTE_USER_HEADER.
+# Doing so is *potentially dangerous*, and it is your responsibility to ensure
+# that only a trusted frontend (usually on the same server) can talk to the
+# redash backend server, otherwise people will be able to login as anyone they
```

Comment:
Yes, that's very true.  I'll edit the comment to that effect.  I'd glossed over that detail since I was using a frontend which always sets the header (overwriting any provided in the original request).  Thanks for the catch!